README.md

```diff
@@ -25,7 +25,13 @@ I know there are a few extensions that already serve this purpose, but most of t
 
 Under the hood, this extension uses the [chrome localStorage API](https://developer.mozilla.org/en/DOM/Storage#localStorage) to store the scroll positions for different webpages. I avoided using sync storage due to its storage limitations ([read more](https://developer.chrome.com/apps/storage)). This extension creates an object which stores the URL as keys and the scroll position as values.
 
-The functions for adding or updating, reading and deleting are in the files `save.js`, `get.js` and `delete.js` respectively, which are executed as content scripts from `popup.js` whenever the respective button is clicked.
+`save.js` handles function for adding/updating the file
```

Comment:
I think you can add a little more detail here on how these functions work because all you have done now is split this paragraph into four separate lines.